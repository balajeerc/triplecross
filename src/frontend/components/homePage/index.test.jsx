import React from 'react';
import renderer from 'react-test-renderer';
import HomePage from './index';

test('Homepage component renders correctly', () => {
  const component = renderer.create(<HomePage />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
