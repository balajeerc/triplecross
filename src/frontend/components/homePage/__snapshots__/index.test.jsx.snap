// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage component renders correctly 1`] = `
<div>
  <h2>
    Hello there!!
  </h2>
  <h3>
    This is a Tiny React Boilerplate
  </h3>
</div>
`;
